### PR TITLE
Java Core and Search September 2020 Release Notes

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: releases/2020-09/java.md [hunk 1]
```diff
@@ -10,15 +10,24 @@ The Azure SDK team is pleased to announce our September 2020 client library rele
...
+- Azure Core Http Netty
+- Azure Core Http OkHttp
+- Azure Search Documents
 
 #### Preview
 
+- Azure Core Experimental
+- Azure Core Serializer Apache Avro
 - Anomaly Detector
+- Azure Search Documents
```
Comment (samvaity) [inline releases/2020-09/java.md:30]: Shouldn't this just be in updates?

File: releases/2020-09/index.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 * Azure Data Tables v2 preview SDK (v12.0.0b1) is released with support for Storage and CosmosDB
 * Azure Schema Registry Beta 1 release
 * Azure Schema Registry Avro Serializer Beta 1 release
+* Azure Cognitive Search SDK for Java added `buildSearchFields` to `SearchIndexClient` and `SearchIndexAsyncClient` to help easily build a search index from a model type.
+* Azure Cognitive Search SDK beta release of document indexing clients.
```
Comment (samvaity) [inline releases/2020-09/index.md:21]: The Serializer packages can have a mention here since they GA'ed?

File: releases/2020-09/java.md (hunk 1 above)
Comment (alzimmermsft) [reply releases/2020-09/java.md:30]: Search is doing an update and beta this month

File: releases/2020-09/index.md (hunk 2 above)
Comment (alzimmermsft) [reply releases/2020-09/index.md:21]: I'll add those

**Review by samvaity — APPROVED** [review]

**Pull request merged**